package.json

```diff
@@ -80,7 +80,7 @@
     "@gnosis.pm/mock-contract": "^3.0.8",
     "@gnosis.pm/owl-token": "^3.1.0",
     "@gnosis.pm/solidity-data-structures": "1.3.5",
-    "@gnosis.pm/util-contracts": "^2.0.6",
+    "@gnosis.pm/util-contracts": "2.0.7",
```

Comment:
I suspect that the `yarn.lock` needs to be re-built becuase we use `--frozen-lockfile` for build.

Reply:
Yes, I needed to wait for the dependent PR to be approved before actually publishing this version.